src/lsstseries/analysis/structure_function/light_curve.py

```diff
@@ -0,0 +1,131 @@
+from typing import Optional
+import numpy as np
+
+MIN_OBSERVATIONS_REQUIRED_FOR_SF = 2
```

Comment:
I think this should be at least 3. If we have 2 datapoints, that is one difference of times, and that has no meaning when we measure variance of one data point.

Reply:
That makes sense, I'll update this.